generate-templates.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 - connection: local
   hosts: 127.0.0.1
+  gather_facts: no
```

Comment:
This should probably go to a separate patch as it applies to Travis as well. It is correct though.

Reply:
done - https://github.com/kubevirt/common-templates/pull/45